content/index.pl.md

```diff
@@ -0,0 +1,252 @@
+-   osobom oganiczonym okolicznościami, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,
+-   osobom korzystającym z wolnego połączenia internetowego albo mającym ograniczony lub kosztowny dostęp do internetu.
+
+Zobacz 7-minutowy film z przykładami tego, jak niezbędna jest dostępność dla osób z niepełnosprawnościami i jak przydatna dla wszystkich w rozmaitych sytuacjach:<br>
+{% include video-link.html title="Oblicza dostępności internetowej <em>(YouTube)</em>" href="https://www.youtube.com/watch?v=3f31oufqFSM" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Więcej o dostępności" class="simple aside" %}
+{:/}
+
+-   Jeśli chcesz dowiedzieć się więcej o tym, jaki wpływ na korzystanie z Internetu mają różne rodzaje niepełnosprawności, lub przeczytać o scenariuszach korzystania z Internetu przez osoby z niepełnosprawnościami, zobacz [Jak osoby z niepełnosprawnościami korzystają z Internetu](/people-use-web/).
+-   Jeśli chcesz poznać więcej przykładów korzyści dla innych, zapoznaj się z materiałami multimedialnymi: [Użytkowanie internetu przez osoby z niepełnosprawnością i bez niej](/media/av/users-orgs/#situations), [Walory dostępności internetowej dla osób z niepełnosprawnością oraz bez niej](https://www.w3.org/WAI/business-case/archive/soc#groups) (materiał archiwalny) oraz archiwalnym materiałem [Wspólne doświadczenia internetowe: Bariery wspólne dla użytkowników urządzeń mobilnych i osób z niepełnosprawnościami](/standards-guidelines/shared-experiences/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Dostępność jest ważna dla osób fizycznych, firm i społeczeństwa {#important}
+
+Internet staje się coraz bardziej istotnym zasobem dla wielu wymiarów życia: edukacji, zatrudnienia, administracji publicznej, biznesu, opieki zdrowotnej, rekreacji i&nbsp;innych. Dostępność Internetu jest niezbędna, aby zapewnić równy dostęp i&nbsp;równe szanse osobom o&nbsp;różnorodnych możliwościach. Dostęp do informacji i komunikacji międzyludzkiej, w tym Internetu, zdefiniowana jest jako podstawowe prawo człowieka w Konwencji Narodów Zjednoczonych o Prawach osób z niepełnosprawnościami (UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)).
+
+Internet oferuje możliwość bezprecedensowego dostępu do informacji i&nbsp;interakcji wielu osobom z&nbsp;niepełnosprawnościami. Oznacza to, że bariery w dostępie do mediów drukowanych, dźwiękowych i&nbsp;wizualnych można znacznie łatwiej pokonać za pomocą technologii internetowych.
+
+Dostępność wspiera włączenie społeczne osób z niepełnosprawnościami, a także innych osób, na&nbsp;przykład:
+
+-   ludzi starszych
+-   ludzi na obszarach wiejskich
+-   ludzi w krajach rozwijających się.
+
+Dostępność internetowa jest w&nbsp;wielu sytuacjach **wymagana przez prawo**.
```

Comment:
A paragraph is missing in the translation :

> There is also a strong business case for accessibility. As shown in the previous section, accessible design improves overall user experience and satisfaction, especially in a variety of situations, across different devices, and for older users. Accessibility can enhance your brand, drive innovation, and extend your market reach.

Reply:
```suggestion
Dostępność internetowa jest w&nbsp;wielu sytuacjach **wymagana przez prawo**.

**Dostępność ma również silne uzasadnienie biznesowe.** Jak pokazano w poprzedniej sekcji, dostępny projekt poprawia ogólne doświadczenie i satysfakcję użytkownika, szczególnie w różnych sytuacjach, na różnych urządzeniach i w przypadku starszych użytkowników. Dostępność może wzmocnić markę, pobudzić innowacje i rozszerzyć zasięg rynkowy.
```